pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
```

Comment:
Further, I don't like the happenstance of combinations. If they ought to all be the same strategy object, there should be a toplevel object exposed. We can make one ourselves I suppose. 

brokerManagementStrategy? Why are they called Strategy anyway?